test/utils.jl

```diff
@@ -81,21 +81,32 @@ module TestUtils
 
             nullarr
             Summary Stats:
+            Mean:           3.500000
+            Minimum:        2.000000
+            1st Quartile:   2.750000
+            Median:         3.500000
+            3rd Quartile:   4.250000
+            Maximum:        5.000000
             Length:         4
-            Type:           Union{Int64, Nulls.Null}
-            Number Unique:  4
+            Type:           Int64
```

Comment:
BTW, in https://github.com/JuliaData/DataTables.jl/pull/68 you used `$Int`: can you do the same here?